### PR TITLE
Update to 4.49.1

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

**Review by gladjohn — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+4.49.1
+==========
+### Bug Fixes
```
Comment (bgavrilMS) [inline CHANGELOG.md:3]:
```suggestion
### New Features

Extended Managed Identity experimental functionality with support for Azure Arc.

### Bug Fixes
```

**Review by bgavrilMS — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (gladjohn) [reply CHANGELOG.md:3]: We did not include that merge commit in the release build we did

Comment (gladjohn) [reply CHANGELOG.md:3]: Ignore the comment as now we are taking the main branch for the release 

@pmaytak FYI

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,14 +15,13 @@
...
+- [Managed identities for Azure resources](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) provide Azure services with an automatically managed identity in Azure Active Directory. You can use this identity to authenticate to any service that supports Azure AD authentication, without having credentials in your code. MSAL now supports acquiring token for managed identities for Azure App Services and Azure Virtual Machines. Use `WithManagedIdentity()` method on the `AcquireTokenForClient` API to get an MSI token. This is an experimental feature and may change in the future versions of MSAL. See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754) and [3829](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3829)
 
 ### Supportability
-- Enabling MSAL Runtime logging integration. See [3575](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3575)
+- Enabled more logging for new WAM broker. See [3575](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3575)
```
Comment (gladjohn) [inline CHANGELOG.md:21]: we need to remove this?

Comment (pmaytak) [reply CHANGELOG.md:21]: I left it, since it does exist in 4.49.0. I'll add a note to 4.49.1 that we disabled it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+4.49.1
+==========
+### New Features
+- Extended managed identity experimental functionality with support for Azure Arc. See [3862](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3862)  
+
+### Bug Fixes
+- Updated the Broker package to use Microsoft.Identity.Client.Interop 0.13.3 to resolve crash related to garbage collection when using new WAM broker preview. See [3868](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3868)
```
Comment (gladjohn) [inline CHANGELOG.md:7]:
```suggestion
- Updated the Broker package to use Microsoft.Identity.Client.NativeInterop 0.13.3 to resolve crash related to garbage collection when using new WAM broker preview. See [3868](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3868)  
```

**Pull request merged**